libs/wingsdk/test/target-tf-aws/schedule.test.ts

```diff
@@ -0,0 +1,85 @@
+import * as cloud from "../../src/cloud";
+import * as std from "../../src/std";
+import * as tfaws from "../../src/target-tf-aws";
+import { Testing } from "../../src/testing";
+import { mkdtemp } from "../../src/util";
+import { tfResourcesOf, tfSanitize, treeJsonOf } from "../util";
+
```

Comment:
Another case worth testing is the expected behavior with multiple schedules, or 1 schedule to multiple functions? Add some assertions about expectations in regards to iam resources maybe. 

We have been trying to get better about corner case coverage in our tests 😅 slowly but surely

Reply:
Nice catch, 1 schedule with multiple roles gave an error due to role id and policy id but I added node.id and hash and now it works